app/locales/en.json

```diff
@@ -7,5 +7,170 @@
+  "Submit report": "Submit report",
+  "Statistics": "Statistics",
+  "Download the dataset as CSV": "Download the dataset as CSV",
+  "COVID-19 infected": "COVID-19 infected",
+  "In total <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> people have reported that they have tested positive for COVID-19 and experience symptoms.": "In total <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> people have reported that they have tested positive for COVID-19 and experience symptoms.",
+  "Symptoms": "Symptoms",
+  "Test results": "Test results",
+  "In total <%= numberWithSpaces(totalPeopleInContactWithInfected) %> people have reported that they have been tested for COVID-19.": "In total <%= numberWithSpaces(totalPeopleInContactWithInfected) %> people have reported that they have been tested for COVID-19.",
+  "Close contacts": "Close contacts",
+  "In total <%= numberWithSpaces(totalPeopleInContactWithInfected) %> people have reported that they have been in close contact with a person who was tested positive for COVID-19.": "In total <%= numberWithSpaces(totalPeopleInContactWithInfected) %> people have reported that they have been in close contact with a person who was tested positive for COVID-19.",
+  "Key figures": "Key figures",
+  "Of those who reported that they have been in close contact with someone that tested positive for COVID-19, have:": "Of those who reported that they have been in close contact with someone that tested positive for COVID-19, have:",
+  "reported that they experience the two most common symptoms those who are infected experience.": "reported that they experience the two most common symptoms those who are infected experience.",
+  "reported that they experience the three most common symptoms those who are infected experience.": "reported that they experience the three most common symptoms those who are infected experience.",
+  "All submissions": "All submissions",
+  "In total <%= numberWithSpaces(totalPeopleWithSymptoms) %> people have reported that they experience symptoms.": "In total <%= numberWithSpaces(totalPeopleWithSymptoms) %> people have reported that they experience symptoms.",
+  "Everyone who has submitted a report, also has:": "Everyone who has submitted a report, also has:",
+  "Total number of reports": "Total number of reports",
+  "Positive test results and close contact": "Positive test results and close contact",
+  "Clarifications data basis": "Clarifications data basis",
+  "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:": "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:",
+  "Symptom": "Symptom",
+  "Added to form": "Added to form",
+  "Headache": "Headache",
+  "Sore throat": "Sore throat",
+  "Loss of sense of taste": "Loss of sense of taste",
```

Comment:
There are some dupes, don't know if that matters or not.

Reply:
That's what happens in merge-hell :( We can remove dupes